lib/sys.h

```diff
@@ -79,10 +79,18 @@ struct metal_init_params {
 	enum metal_log_level		log_level;
 };
 
+#ifndef METAL_INIT_LOG_HANDLER
+#define METAL_INIT_LOG_HANDLER metal_default_log_handler
+#endif
+
+#ifndef METAL_INIT_LOG_LEVEL
+#define METAL_INIT_LOG_LEVEL METAL_LOG_INFO
+#endif
```

Comment:
No reason, did it for consistency.

Reply:
How about in the system/zephyr/sys.h
#define  METAL_INIT_DEFAULTS = {  ...} 

lib/sys.h
move the `#define METAL_INIT_DEFAULTS` after 
```
#include <metal/system/@PROJECT_SYSTEM@/sys.h>
```
and it will look like:
```
#ifndef METAL_INIT_DEFAULTS
#define METAL_INIT_DEFAULTS {...}
#endif
```

Reply:
Sounds, good.  Made the changes that you suggested.